atom/common/node_bindings_win.cc

```diff
@@ -11,7 +11,16 @@
 namespace atom {
 
 NodeBindingsWin::NodeBindingsWin(BrowserEnvironment browser_env)
-    : NodeBindings(browser_env) {}
+    : NodeBindings(browser_env) {
+  // on single-core the io comp port NumberOfConcurrentThreads needs to be 2
+  // to avoid cpu pegging likely caused by a busy loop in PollEvents
+  char* numProcessors = getenv("NUMBER_OF_PROCESSORS");
```

Comment:
This isn't the right way to test whether we're single-core. Use `base::SysInfo::NumberOfProcessors()`, declared in `base/sys_info.h`.

Reply:
But it makes for easy testing ;)